tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py

```diff
@@ -329,7 +329,7 @@ def _update_step_fn(var, grad):
           with ops.control_dependencies(_before):
             _apply_op = _update_step_fn(var, grad)
 
-          with ops.control_dependencies([_apply_op]):
+          with ops.control_dependencies([var]):
```

Comment:
At new Keras 3 optimizer, there's not returning object from OPT update_step function. It just a multiple operators combination for XLA jit compiling. So _apply_op will be a None when use Keras>=2.13 default optimizer.

Reply:
This is related to the update logic of TFRA. var variable(TrainableWrapper) is a training agent for TF. 
Because TFRA does not have its own optimizer but reuses TF's native optimizer, TF cannot directly update the values in the KV table like other open source projects. Instead, TF will update a temporary variable(TrainableWrapper). After the gradient is updated to var, the new values of var will be inserted back into KV table.

Reply:
> 👍🏻 thanks for the info. one last question, remove `_apply_op =` from below seems valid? with ops.control_dependencies(_before): _apply_op = _update_step_fn(var, grad)

Yes, of course.